src/workflows-app/utils/submission-utils.test.js

```diff
@@ -402,6 +402,58 @@ describe('validateInputs', () => {
       ])
     );
   });
+
+  it('should display info and error messages for record lookup type coercions and errors', () => {
+    const dataTableAttributes = {
+      singleNum: {
+        datatype: 'NUMBER',
+      },
+      arrayOfString: {
+        datatype: 'ARRAY_OF_STRING',
+      },
+    };
+
+    const inputsWithRecordLookups = [
+      {
+        ...arrayInput('validInt', 'Int'), // success with info
+        source: {
+          type: 'record_lookup',
+          record_attribute: 'singleNum',
+        },
+      },
+      {
+        ...arrayInput('invalidInt', 'Int'), // failure
+        source: {
+          type: 'record_lookup',
+          record_attribute: 'arrayOfString',
+        },
+      },
```

Comment:
For completeness, do you also want to have a completely valid array input that you don't expect to show up in either of the error or info lists below?